etc/DependencyInstaller.sh

```diff
@@ -332,6 +352,10 @@ case "${os}" in
 To enable GCC-8 or Clang-7 you need to run:
     source /opt/rh/devtoolset-8/enable
     source /opt/rh/llvm-toolset-7.0/enable
+if you have provided a prefix, set cmake as an alias:
+    alias cmake='<YOUR-PREFIX>/bin/cmake' 
+    or  run
+    source ~/.bash_aliases
```

Comment:
we know if the user has used the argument, or at least we can create a flag for that and print a conditional message here. also, we can use the actual prefix in the message instead of the `<YOUR_PREFIX>` -> `$PREFIX`.